docs/rfcs/accepted/0002-consolidate_svp_and_non_svp_cipher_apis/README.md

```diff
@@ -16,8 +18,6 @@ to the SecApi 3 library. The intention is that the SecApi 2 Adapter library can
 with a new SecApi 3 library and can provide compatability for applications and libraries that have
 not upgraded to the new interface yet.
 
-![SecApi3-Plan](../diagrams/SecApi3-Plan.png)
```

Comment:
I thought this was being added back in.

Reply:
OK, I will add it back but I have to move it out of the diagram directory into this directory.

Reply:
Done